### DISCUSSION TITLE
Should pytest.raises handle PEP-654 ExceptionGroups?

### DISCUSSION THREAD

**Discussion by b0g3r** [pytest-dev/pytest#11437] [Ideas] [answered]
If I have a function which raises `ExceptionGroup`, how should I use it with `pytest.raises`?

Typical scenario without groups will pass:

```python3
with pytest.raises(ValueError):
  raise ValueError()
```

But with groups it will fail with uncatched ExceptionGroup:

```python3
with pytest.raises(ValueError):
  raise ExceptionGroup('problem', [ValueError()])
```

```
  |     raise ExceptionGroup('problem', [ValueError()])
  | ExceptionGroup: problem (1 sub-exception)
  +-+---------------- 1 ----------------
    | ValueError
    +------------------------------------
```

Maybe pytest should support exception groups in `raises`?

**Comment by nicoddemus** [comment, marked as answer]
There's an ongoing discussion in #10441.

**Comment by RonnyPfannschmidt** [comment]
Personally I'm opposed to lerting raises handle exception groups due to the difference in cardinality
